docs/Gemfile

```diff
@@ -36,3 +36,5 @@ gem "wdm", "~> 0.1.0", :install_if => Gem.win_platform?
 # kramdown v1, comment out this line.
 gem "kramdown-parser-gfm"
 
+
+gem "webrick", "~> 1.7"
```

Comment:
what's this for?

Reply:
With Ruby 3, webrick is no longer part of standard libraries and it needs to be installed.
Refer this - https://github.com/jekyll/jekyll/issues/8523

Based on @jmg-duarte comments(https://github.com/foambubble/foam/pull/581#issuecomment-819673827), I'll remove this webrick addition. Hope this is fine.